hub-config/tasks.json

```
@@ -3,17 +3,186 @@
+                                "2024-03-02", "2024-03-09", "2024-03-16", "2024-03-23", 
+                                "2024-03-30", "2024-04-06", "2024-04-13", "2024-04-20", 
+                                "2024-04-27", "2024-05-04", "2024-05-11", "2024-05-18",
+                                "2024-05-25", "2024-06-01",
+                                "2024-11-09", "2024-11-16", "2024-11-23", "2024-11-30", 
+                                "2024-12-07", "2024-12-14", "2024-12-21", "2024-12-28", 
+                                "2025-01-04", "2025-01-11", "2025-01-18", "2025-01-25", 
+                                "2025-02-01", "2025-02-08", "2025-02-15", "2025-02-22", 
+                                "2025-03-01", "2025-03-08", "2025-03-15", "2025-03-22", 
+                                "2025-03-29", "2025-04-05", "2025-04-12", "2025-04-19", 
+                                "2025-04-26", "2025-05-03", "2025-05-10", "2025-05-17", 
+                                "2025-05-24", "2025-05-31", "2025-06-07", "2025-06-14",
+                                "2025-06-21"
+
+                            ]
                         }
                     },
-                    "output_type": {},
-                    "target_metadata":[]
+                    "output_type": {
+                        "quantile": {
+                            "output_type_id": {
+                                "required": [
+                                    0.01,
+                                    0.025,
+                                    0.05,
+                                    0.1,
+                                    0.15,
+                                    0.2,
+                                    0.25,
+                                    0.3,
+                                    0.35,
+                                    0.4,
+                                    0.45,
+                                    0.5,
+                                    0.55,
+                                    0.6,
+                                    0.65,
+                                    0.7,
+                                    0.75,
+                                    0.8,
+                                    0.85,
+                                    0.9,
+                                    0.95,
+                                    0.975,
+                                    0.99
+                                ],
+                                "optional": null
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0
+                            }
+                        },
+                        "sample": {
+                            "output_type_id_params": {
+                                "is_required": false,
+                                "type": "integer",
```

There is a small difference here compared to the current FluSight hub: 

```suggestion
                                "type": "character",
                                "max_length": 15,
```